catalog/imaging/imaging-transformer-nitf/src/test/java/org/codice/alliance/transformer/nitf/gmti/TestNitfGmtiTransformer.java

```diff
@@ -95,75 +100,16 @@ public void testTre() throws IOException, CatalogTransformerException, NitfForma
         assertThat(metacard.getMetacardType()
                 .getName(), is("gmti"));
 
-        assertThat(metacard.getAttribute(MtirpbAttribute.AIRCRAFT_ALTITUDE.getAttributeDescriptor()
-                .getName())
-                .getValue(), is("150000"));
+
         assertThat(metacard.getAttribute(MtirpbAttribute.NUMBER_OF_VALID_TARGETS.getAttributeDescriptor()
                 .getName())
                 .getValue(), is("001"));
-        assertThat(metacard.getAttribute(MtirpbAttribute.AIRCRAFT_ALTITUDE_UNITS.getAttributeDescriptor()
-                .getName())
-                .getValue(), is("m"));
-        assertThat(metacard.getAttribute(MtirpbAttribute.AIRCRAFT_HEADING.getAttributeDescriptor()
-                .getName())
-                .getValue(), is("000"));
         assertThat(metacard.getAttribute(MtirpbAttribute.AIRCRAFT_LOCATION.getAttributeDescriptor()
                 .getName())
                 .getValue(), is("+52.123456-004.123456"));
```

Comment:
When I run this locally, I see this:

java.lang.AssertionError: 
Expected: is "+52.123456-004.123456"
     but: was "POINT (+52.123456 -004.123456)"
Expected :+52.123456-004.123456
Actual   :POINT (+52.123456 -004.123456)